From 24944942ce022bb4feb8ab32a49d9c9e8438f09d Mon Sep 17 00:00:00 2001
From: David Holsgrove <david.holsgrove@petalogix.com>
Date: Tue, 18 Oct 2011 10:37:41 +1000
Subject: [PATCH 09/17] elf-gloss-linux.specs: Switch lxil to lgloss-linux for dejagnu testing

as we now output libxil.a not libgloss.a from the xilinx merged newlib/libgloss build,
and add lxil-linux instead of lxil(Edgars minimual implementation) to LIB_SPEC for
for dejagnu testing

Signed-off-by: David Holsgrove <david.holsgrove@petalogix.com>
---
 libgloss/microblaze/elf-gloss-linux.specs |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libgloss/microblaze/elf-gloss-linux.specs b/libgloss/microblaze/elf-gloss-linux.specs
index 1674622..cedf2db 100644
--- a/libgloss/microblaze/elf-gloss-linux.specs
+++ b/libgloss/microblaze/elf-gloss-linux.specs
@@ -4,11 +4,11 @@
 *link:
 %(old_link) -defsym _TEXT_START_ADDR=0x10000 -z max-page-size=4096 -z common-page-size=4096 --no-omagic
 
-*libgloss:
+*libxil:
 -lgloss-linux
 
 *lib:
--start-group -lgloss-linux -lxil -lc -lm -end-group
+-start-group -lgloss-linux -lxil-linux -lc -lm -end-group
 
 *startfile:
 linux-crt0%O%s crti%O%s crtbegin%O%s
-- 
1.7.1

